### PR TITLE
add readme about logger

### PR DESCRIPTION
**Pull request by ardityawahyu**
just adding readme about the logger

### CODE REVIEW

**Review by lukluk — APPROVED** [review]
brilliant, awesome @ardityawahyu

**Review by zackijack — APPROVED** [review]
LGTM 🍰

File: README.md [hunk 1]
```diff
@@ -16,4 +18,21 @@ $ go get github.com/kitabisa/perkakas/v2
 
 ## How To Use
 
-See `README.md` in each folders to see how to use these modules.
\ No newline at end of file
+See `README.md` in each folders to see how to use these modules.
+
+### Logger
+
+nb: logger from is deprecated.
```
[the remark this answers is not in the document]
Comment (ardityawahyu) [reply README.md]: ho iya, updated mas @budiariyanto

**Review by budiariyanto — APPROVED** [review]
LGTM :+1:

**Pull request merged**